server/security/security.steps.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { Fusion } from 'jest-cucumber-fusion';
+import 'test_common/commonServerSteps';
```

Comment:
this should not be needed - it should be provided for you

Reply:
I thought as much, but it doesn't - https://github.com/b-yond-infinite-network/jest-cucumber-fusion/blob/master/docs/ReusingStepDefinitions.md suggests the same

Reply:
I am very curious how any of the other tests work in this case. These steps should be preloaded for everyone here: https://github.com/strimzi/strimzi-ui/blob/ae571ee62875127d08716a42beb1ba9f58f79651/test_common/jest_cucumber_support/index.ts / https://github.com/strimzi/strimzi-ui/blob/74f9304bfd32ad7008c3f6eefd2bef8907a5d2cc/test_common/jest.common.config.js . If you want/need to interact with the world then it make sense to import the functions to be able to do that.

Reply:
Raised issue - the existing tests work because they import the world and thus trigger the embedded given/when/then step definitions